draft-demarco-oauth-status-attestations.md

```diff
@@ -368,9 +372,27 @@ Status Assertion,
 MUST demonstrate the proof of possession of the Digital Credential to
 the Credential Issuer.
 
-The proof of possession is made by enclosing the Digital Credential in an
-object (JWT) signed with the private key that its related public key is
-referenced in the Digital Credential.
+The proof of possession is made by enclosing the Digital Credential
+in an object (JWT or CWT) using the same confirmation method referenced in RFC7800.
```

Comment:
the digital credential is not enclosed, an hash of it is provided in the status assertion request and the same hash in the status assertion response